Add sentence for second person to style guide (#865)
grafana/writers-toolkit: docs/sources/write/style-guide/style-conventions/index.md

## Address users clearly

> Our style prefers second person ("you") and avoids first person ("I"/"we").
> 
> To address users clearly and directly, write directives in the imperative second person where the unspoken 'you' is implied.
> 
> Avoid making the user or their role, such as _system administrator_, the subject of a sentence, as in: "Users configure the Cloud using a secure shell.".

Author: ```suggestion
<!-- vale Grafana.We = NO -->
To address users clearly and directly, use second person ("you") and avoid first person ("I" or "we"). 
Write directives in the imperative second person where the unspoken 'you' is implied.
<!-- vale Grafana.We = Yes -->

Avoid making the user or their role, such as _system administrator_, the subject of a sentence, as in: "Users configure the Cloud using a secure shell.".
```

Reviewer: That sounds good to me!

## Address users clearly

<!-- vale Grafana.We = NO -->
To address users clearly and directly, use second person ("you") and avoid first person ("I", "our", or "we"). 
Use first person plural pronouns like 'our' carefully.
Don't use 'we' when you're talking about the reader, instead use 'you'.

> It's OK to use 'we' when you're talking about Grafana Labs.

Author: ```suggestion
You can use 'we' when you're talking about Grafana Labs.
```

> <!-- vale Grafana.We = Yes -->

Reviewer: ```suggestion
<!-- vale Grafana.We = YES -->
```
